website/sidebars.json

```diff
@@ -74,7 +74,7 @@
           "roles/developer/contracts/cli",
           "roles/developer/contracts/rust",
           "development/calling-smart-contracts",
-          "roles/developer/contracts/rpc"
+          "roles/developer/contracts/api"
```

Comment:
Reminder: we need to have proper redirects when we move pages

How is that configured? Is it completely on Render side?

Reply:
On it! Got a list of URL forwardings that will need to take place. And yes, Render is currently handling all of this.